application/src/main/resources/configs.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
What do these empty files give us?

Reply:
These files need to be in a specific directory for the app to access them and I thought it would be easier to just find and edit them rather than make a new file and have to recall where it needs to go and what it needs to be named. We could also add properties so you can configure where these files are located but since reading configs and sites from json files is only going to be used in the first iteration(s) I didn't bother.